src/sentry/auth/system.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import absolute_import, print_function
+
+import ipaddress
+import six
+
+from django.conf import settings
+from django.contrib.auth.models import AnonymousUser
+from django.utils.crypto import constant_time_compare
+from uuid import uuid4
+
+from sentry import options
+from sentry.utils.cache import memoize
+
+
+INTERNAL_NETWORKS = [
```

Comment:
Can you also update the code in the superuser code to use the same instances? https://github.com/getsentry/sentry/blob/d292fd4680274bf638ffb73fa5584593bd0642f7/src/sentry/auth/superuser.py#L61-L63

Reply:
That's using a different setting though, `SUPERUSER_ALLOWED_IPS`. Still?

Reply:
@jan-auer the fallback for `SUPERUSER_ALLOWED_IPS` is `INTERNAL_IPS`.

Reply:
Actually for our own case it does not matter because we permit superuser from any ip and use other systems to secure it.